auth_backends/backends.py

```diff
@@ -290,6 +292,16 @@ def auth_complete_params(self, state=None):
         params['token_type'] = 'jwt'
         return params
 
+    def auth_complete(self, *args, **kwargs):
+        """
+        Override this method to emit the `EdXOAuth2.auth_complete_signal` signal.
```

Comment:
Doesn't this method emit the mentioned signal below? One doesn't need to override the method to emit the signal - it already gets sent, correct?

Reply:
The wording could be clearer, I'll update it to say "We override this method to...".